insights/templates/base.html.j2

```diff
@@ -15,14 +15,11 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.bundle.js"></script>
   <script src="https://unpkg.com/vue-chartjs/dist/vue-chartjs.js"></script>
   {% else %}
-  <script src="https://cdn.jsdelivr.net/npm/vue@2"></script>
+  <script src="https://cdn.jsdelivr.net/npm/vue@2.6.14/dist/vue.min.js" integrity="sha256-kXTEJcRFN330VirZFl6gj9+UM6gIKW195fYZeR3xDhc=" crossorigin="anonymous"></script>
```

Comment:
Is this correct?

Reply:
Yes looks correct to me, what is the concern?